releasenotes/notes/python39-617f3ed823e63cfe.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Python 3.9 support has been added in this release. You can now run Qiskit
+    Terra using Python 3.9. 
```

Comment:
Can you add a comment that, at time of release, users who want to use matplotlib will need to build it from source? Then I think this is good to go.

Reply:
Heh, I just started to write this. But it looks like there was a 3.3.3 release ~12 hours ago that added 3.9 wheels: https://pypi.org/project/matplotlib/#files